fake_switches/brocade/command_processor/config_virtual_interface_vrrp.py

```diff
@@ -16,12 +16,9 @@
 
 
 class ConfigVirtualInterfaceVrrpCommandProcessor(ConfigInterfaceCommandProcessor):
-    def __init__(self, switch_configuration, terminal_controller, logger,
-                 piping_processor, port, vrrp):
-        super(ConfigVirtualInterfaceVrrpCommandProcessor, self).__init__(
-            switch_configuration, terminal_controller, logger, piping_processor,
-            port)
-        self.vrrp = vrrp
+    def init(self, switch_configuration, terminal_controller, logger, piping_processor, *args):
+        super(ConfigVirtualInterfaceVrrpCommandProcessor, self).init(switch_configuration, terminal_controller, logger, piping_processor, args[0])
```

Comment:
Actually the inheritance to the interface processor made no sense as the VRRP processor do not share the commands... so i'll change the parent instead and the params assign becomes

        self.port, self.vrrp = args